models/human.rb

```diff
@@ -25,5 +25,10 @@ def job_title=(value)
   def job_title
     @job_title
   end
-end
+  attr_accessor :age
+
+  def can_drink?
```

Comment:
Great simple implementation..
What happens if this is called and `@age` was never set? (we will discuss next week)